machine/delta/delta_ag9032v1/busybox/patches/series

```diff
@@ -0,0 +1,2 @@
+onie-sys-eeprom-platform.patch
+onie-sys-eeprom-write-protection.patch
```

Comment:
When you have a common patch that applies to multiple machines you can put the patch one level up in the <vendor>/<machine> directory so that all machines can share the same patch.  This is better than make several copies of a patch.

This can be done for both the kernel and busybox.  For example put your patches here:
```
machine/delta/busyboxy
machine/delta/kernel
```

See how accton has done this:
https://github.com/opencomputeproject/onie/tree/master/machine/accton